easybuild/easyconfigs/p/Perl-bundle-CPAN/Perl-bundle-CPAN-5.36.1-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,1885 @@
+easyblock = 'Bundle'
+
+name = 'Perl-bundle-CPAN'
+version = '5.36.1'
+
+homepage = 'https://www.perl.org/'
+description = """A set of common packages from CPAN"""
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('binutils', '2.40'),
+    ('groff', '1.22.4'),
+]
+
+dependencies = [
+    ('Perl', version),
+    ('zlib', '1.2.13'),  # for Net::SSLeay
+    ('expat', '2.5.0'),  # for XML::Parser
+    ('ncurses', '6.4'),  # for Term::ReadLine::Gnu
+    ('libreadline', '8.2'),  # for Term::ReadLine::Gnu
+    ('OpenSSL', '1.1', '', SYSTEM),  # required for Net::SSLeay
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+# !! order of extensions is important !!
+# extensions updated on 2023-05-23
+exts_list = [
+    ('Config::General', '2.65', {
+        'source_tmpl': 'Config-General-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/T/TL/TLINDEN'],
+        'checksums': ['4d6d5754be3a9f30906836f0cc10e554c8832e14e7a1341efb15b05d706fc58f'],
+    }),
+    ('File::Listing', '6.15', {
+        'source_tmpl': 'File-Listing-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/P/PL/PLICEASE'],
+        'checksums': ['46c4fb9f9eb9635805e26b7ea55b54455e47302758a10ed2a0b92f392713770c'],
+    }),
+    ('ExtUtils::InstallPaths', '0.012', {
+        'source_tmpl': 'ExtUtils-InstallPaths-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/L/LE/LEONT'],
+        'checksums': ['84735e3037bab1fdffa3c2508567ad412a785c91599db3c12593a50a1dd434ed'],
+    }),
+    ('ExtUtils::Helpers', '0.026', {
+        'source_tmpl': 'ExtUtils-Helpers-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/L/LE/LEONT'],
+        'checksums': ['de901b6790a4557cf4ec908149e035783b125bf115eb9640feb1bc1c24c33416'],
+    }),
+    ('Test::Harness', '3.44', {
+        'source_tmpl': 'Test-Harness-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/L/LE/LEONT'],
+        'checksums': ['7eb591ea6b499ece6745ff3e80e60cee669f0037f9ccbc4e4511425f593e5297'],
+    }),
+    ('ExtUtils::Config', '0.008', {
+        'source_tmpl': 'ExtUtils-Config-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/L/LE/LEONT'],
+        'checksums': ['ae5104f634650dce8a79b7ed13fb59d67a39c213a6776cfdaa3ee749e62f1a8c'],
+    }),
+    ('Module::Build::Tiny', '0.045', {
+        'source_tmpl': 'Module-Build-Tiny-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/L/LE/LEONT'],
+        'checksums': ['d20692eee246d0b329b7f7659f36286b14ae0696fe91078a64b7078f8876d300'],
+    }),
```

Comment:
This is what the build fails on, Module::Build::Tiny needs ExtUtils::Config. I guess we have to prepend a PERLPATH thing in order for this to be picked up.

Reply:
or maybe rather a custom PerlBundle that does this, and sanity paths

Reply:
I swapped to a new easyblock, though it still needs work